x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/_index.scss

```diff
@@ -1 +1 @@
-@import 'analytics_table';
+@import 'analytics_table';
```

Comment:
please revert this change, it unnecessary triggers the kibana-design team

Reply:
Reverted this in https://github.com/elastic/kibana/pull/66934/commits/d9342f6c5a5616ee500d34b1f251ef22affa0d45